html/captive-portal/content/billing/authorizenet.js

```diff
@@ -31,7 +30,7 @@
               for (var i = 0; i < response.messages.message.length; i++) {
                   console.log(response.messages.message[i].code + ": " + response.messages.message[i].text);
               }
-              var $form = $('#payment-form');
+              var $form = document.getElementById('payment-form');
               $form.find('.payment-errors p').text('Unable to proceed with payment please contact your service provider');
```

Comment:
A very newb question. Back when I did JS, you couldn't use very advanced functions on its result (like find/text/etc). Just double checking that the find method (and text) are indeed defined for the object returned by getElementById now

Reply:
`.find()` in this context is a jQuery function that was missed. I'll revisit.